src/Aquifer.Public.API/Endpoints/Bibles/List/Endpoint.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Aquifer.Data;
+using FastEndpoints;
+using Microsoft.EntityFrameworkCore;
+
+namespace Aquifer.Public.API.Endpoints.Bibles.List;
+
+public class Endpoint(AquiferDbContext dbContext) : Endpoint<Request, IReadOnlyList<Response>>
+{
+    private readonly AquiferDbContext _dbContext = dbContext;
```

Comment:
Don't think we need this. Can just use `dbContext` throughout.

Reply:
See https://github.com/BiblioNexusStudio/aquifer-server/pull/443/files#r1765532162 for discussion on this.